### PR TITLE
Add Visual Studio 2019 support & tools updates

### PR DESCRIPTION
**Pull request by djipi**
Added a Visual Studio 2019 projects for 64 bits.
Added short text descriptions, and usages.
Fix some typos.
Modified some texts display.
Added PI definition in case of it is missing in the headers.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,19 @@
-tga2cry
 tga2cry.exe
-tgainfo
```
Comment (cubanismo) [inline .gitignore]: Why remove these?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# TGA, RGB and CRY tools
+A set of tools originaly developed for PC/MSDOS.<br>
+Adapted for Visual Studio 2019 64 bits.
```
Comment (cubanismo) [inline README.md]: "Adapted for Linux (32/64-bit x86)  and Visual Studio 2019 (64-bit x86)"

File: rgb2cry.c [hunk 3]
```diff
@@ -1,27 +1,37 @@
...
+#include <stdlib.h>
 
 typedef unsigned short UWORD;
 typedef unsigned char UBYTE;
 typedef short WORD;
 typedef char BYTE;
 
 UWORD do_cry(int, int, int);
-extern int atoi(char *);
+//extern int atoi(char *);
```
Comment (cubanismo) [inline rgb2cry.c]: Just delete it rather than commenting it out.  It's still there in git history if someone needs to dig it up.

File: scale.c [hunk 4]
```diff
@@ -28,8 +28,12 @@
 #endif
 
 #ifndef M_PI
+#ifndef PI
+#define M_PI 3.14159265358979323846
```
Comment (cubanismo) [inline scale.c]: I recall running in to this issue on windows in the past as well.  Apparently defining math constants in math.h is not strictly ANSI C, so MS doesn't include them by default.  Defining _USE_MATH_DEFINES before including math.h will avoid the need to manually define it with Visual C++ though.

**Review by cubanismo — CHANGES_REQUESTED** [review]
Haven't had time to test this one myself yet, but a few suggestions inline to start.

File: scale.c (hunk 4 above)
Comment (djipi) [reply scale.c]: Fixed the issue with your recommendation.

File: README.md (hunk 2 above)
Comment (djipi) [reply README.md]: Replaced with your line.

File: rgb2cry.c (hunk 3 above)
Comment (djipi) [reply rgb2cry.c]: This is good point, fixed.

**Comment by cubanismo** [comment]
Tested the latest version out, and everything looks good.  If you restore the lines removed from .gitignore, I'll merge this.

**Comment by djipi** [comment]
Fixed the .gitignore

**Comment by cubanismo** [comment]
Thanks, I see what you were trying to do now with .gitignore.  I made some adjustments to the series and pushed that version instead.  Closing this, please rebase on top of my branch and let me know if I missed anything/introduced any issues.

**Pull request closed without merging**